example/deployments/cumulus/sandbox.tfvars

```diff
@@ -52,5 +52,4 @@ ems_username          = "cumulus"
 
 thin_egress_jwt_secret_name = "cumulus_sandbox_jwt_tea_secret"
 
-include_orca = false
```

Comment:
This was true before, is there a reason you're deleting it instead of changing it back?

Reply:
The default include_orca is true. No need to specify.